docs/TroubleShooting.md

```diff
@@ -54,6 +54,22 @@ make
 sudo make install
 ```
 
+### Issues with manual toolchain setup
+When ```make ``` executes, you may encounter below error:
+```
+arm-none-eabi-gcc: Command not found
+```
+This may happen if the toolchain path is not added to ```PATH```. To resolve:
+```
+export PATH=<Your Toolchain PATH>:$PATH
+```
+If toolchain path is added to ```PATH```, then this may be because the toolchain you 
+set uses 32-bit libs on the 64-bit system. For this matter, install the below toolchain 
+as alternative.  To resolve:
```

Comment:
Two spaces make new line for md viewer, but not this editor format.
So it is better to add enter also for readability.

Reply:
Thanks Mr. Kim, I have moved "To resolve:" into a new line in both places.